server/src/main/java/com/defold/extender/ExtenderController.java

```diff
@@ -42,7 +42,6 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.net.URISyntaxException;
```

Comment:
Here and below - remove `URISyntaxException` because it's thrown any more from `DefoldSdkService::getRemoteSdk`